js/controller.js

```diff
@@ -115,7 +115,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "forceControlBarVisible": false,
       "timer": null,
       "errorCode": null,
-      "isSubscribed": false,
+
+      "isSubscribed": {
```

Comment:
Almost there. But I think we can have 2 separate flags. That means you do not need to create an Object out of isSubscribed, and make isSubscribed as-is.